src/index.spec.js

```diff
@@ -170,7 +177,39 @@ var serviceWorkerOption = {
             )
           }
         )
-      })
+      }),
+        it('should get passed assetsHash', done => {
+          const transformOptions = serviceWorkerOption => {
+            expect(serviceWorkerOption)
+              .to.have.property('assetsHash')
+              .that.is.a('string')
```

Comment:
As the input is known, so is the output - could you update the test case and let it test against the actual expected value of the hash?